uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -318,6 +318,122 @@ define([
           expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ activity_date_time: '' }));
         });
       });
+
+      describe('document contains files', function () {
```

Comment:
I am trying to read tests and I am not quite sure I understand the difference between `document contains files` and `document already has attachments`. Could you clarify please?